daniel-aguirre-salamanca/tarea-5/helpers.js

```diff
@@ -0,0 +1,31 @@
+function changeClass(button) {
+  const number = parseInt(button.textContent);
+  if (button.id === "empty") {
+    button.className = "empty";
+  } else if (number % 2 === 0) {
+    button.className = "active active--even";
+  } else {
+    button.className = "active active--odd";
+  }
+}
+
+function generateRandomness() {
+    let array = [1,2,3,4,5,6,7,8,9,10,11,12,13,14,15];
+    for (let i = array.length - 1; i > 0; i--) {
```

Comment:
@daguirreglobant , please use modern approach like forEach, map or reduce, also remember a proper naming for variable i